docs/kube_play_support.md

```diff
@@ -1,6 +1,6 @@
-# Podman Play Kube Support
+# Podman Kube Play Support
```

Comment:
Should we rename the file to just `kubernetes_support.md`?

@rhatdan @umohnani8 @baude

Reply:
yup, rename sounds good to me.